### DISCUSSION TITLE
Build status page

### DISCUSSION THREAD

**Discussion by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1414] [Ideas]
Just wanted to record Dave's idea in an issue so we don't forget about it, because I think it'd be a great addition to the site:

Have a page where we show and link to recently completed builds and their status. This should drive home the point all the machinery that's at work behind the scenes and it could also serve as a way to get an idea where we're at with respect to processing outside of monitoring.

**Comment by daveverwer** [comment]
I keep thinking about this too. I even had a fever dream where the page showed all the build machines represented by illustrations of factories at the top of the page with varying amounts of smoke coming out of their chimneys depending on their load.

Or, you know, we could put the builds in a list. 😂
